content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -808,6 +808,40 @@ Example: `node.cloudprovider.kubernetes.io/shutdown: "NoSchedule"`
 
 If a Node is in a cloud provider specified shutdown state, the Node gets tainted accordingly with `node.cloudprovider.kubernetes.io/shutdown` and the taint effect of `NoSchedule`.
 
+{{< note >}}
+Node Feature Discovery(NFD) is a Kubernetes add-on for detecting hardware features and
+system configuration. To learn more about NFD and its components go to its official
+[documentation](https://kubernetes-sigs.github.io/node-feature-discovery/v0.7/get-started/).
+{{< /note >}}
+
+### nfd.node.kubernetes.io/master.version
+
+Example: `nfd.node.kubernetes.io/master.version: v0.6.0`
+
+This annotation defines version of the NFD-Master instance running on the node.
+It is used for informative use only.
```

Comment:
It'd be better if we clarify who is using it.

Reply:
Hi @divya-mohan0209 , these annotations are under the list where it's mentioned that:
`The taints listed below are always used on Nodes
`
it's [here](https://kubernetes.io/docs/reference/labels-annotations-taints/#node-kubernetes-io-not-ready)

Reply:
I meant that in the sentence you've mentioned that these annotations are used for informative purposes only. It'd help clarify what is using these annotations. Is it NFD that's using them?

Reply:
The master.version is only created on nodes running nfd-master. so yes it is used for NFD mainly.

Reply:
I can mention to each , 
`Used on: Nodes running NFD`
WDYT @divya-mohan0209 ?

Reply:
`Used on: Nodes`

is right. In the text, mention that it only applies for nodes where NFD is running.